<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Derivatives</class>
 <widget class="QDialog" name="Derivatives">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>826</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>10</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Derivatives Lab</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>760</y>
     <width>861</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Delta</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>290</y>
       <width>0</width>
       <height>321</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNum_prob">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>750</y>
       <width>40</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gamma</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Vega</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Theta</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Rho</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei UI</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#aa0000;&quot;&gt;TaiwanTech Derivatives Lab&lt;/span&gt;&lt;span style=&quot; font-size:14pt; color:#aa0000; vertical-align:super;&quot;&gt;TM&lt;/span&gt;&lt;span style=&quot; font-size:14pt; color:#313194;&quot;&gt;    Option Pricing and Strategies 選擇權定價與策略 Module 3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>780</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QSlider" name="slider_t2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>860</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_t1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>830</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>830</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>860</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_t1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>820</y>
     <width>114</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_t2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>850</y>
     <width>114</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>The Greek Letters</string>
   </property>
   <widget class="QRadioButton" name="radioDelta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delta</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioGamma">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamma</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioVega">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vega</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioTheta">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Theta</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioRho">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rho</string>
    </property>
   </widget>
  </widget>
  <widget class="MPL_WIDGET" name="oMplCanvas" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>921</width>
     <height>701</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="slider_r">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>110</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider_sigma">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>190</y>
     <width>140</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
   <property name="sliderPosition">
    <number>20</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider_q">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>150</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider_K">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>70</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>80</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="sliderPosition">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider_slope">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>270</y>
     <width>140</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>No. of paths to be shown</string>
   </property>
   <property name="minimum">
    <number>20</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>85</number>
   </property>
   <property name="sliderPosition">
    <number>85</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_r">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>r = </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sigma">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>σ =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_q">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>140</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>q =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_K">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>K = </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_slope">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>S0 = </string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_T">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>230</y>
     <width>140</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="sliderPosition">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_T">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>T= 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_reset">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>750</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset to default parameters</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_Taylor">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>320</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Show Taylor polynomial</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_sigma_2">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(at green line)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_Taylor1">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>360</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear (1st order)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_Taylor2">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>390</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Quadratic (2nd order)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_Theta_T2M">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>450</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Theta vs. time to maturity</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MPL_WIDGET2" name="oMplCanvas_Theta_T2M" native="true">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>490</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>label</zorder>
  <zorder>buttonBox</zorder>
  <zorder>slider_t2</zorder>
  <zorder>slider_t1</zorder>
  <zorder>label_9</zorder>
  <zorder>label_11</zorder>
  <zorder>label_t1</zorder>
  <zorder>label_t2</zorder>
  <zorder>groupBox</zorder>
  <zorder>slider_r</zorder>
  <zorder>slider_sigma</zorder>
  <zorder>slider_q</zorder>
  <zorder>slider_K</zorder>
  <zorder>slider_slope</zorder>
  <zorder>label_r</zorder>
  <zorder>label_sigma</zorder>
  <zorder>label_q</zorder>
  <zorder>label_K</zorder>
  <zorder>label_slope</zorder>
  <zorder>slider_T</zorder>
  <zorder>label_T</zorder>
  <zorder>button_reset</zorder>
  <zorder>check_Taylor</zorder>
  <zorder>label_sigma_2</zorder>
  <zorder>check_Taylor1</zorder>
  <zorder>check_Taylor2</zorder>
  <zorder>check_Theta_T2M</zorder>
  <zorder>oMplCanvas</zorder>
  <zorder>oMplCanvas_Theta_T2M</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPL_WIDGET</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MPL_WIDGET2</class>
   <extends>QWidget</extends>
   <header location="global">mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Derivatives</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Derivatives</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
